core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -75,8 +75,6 @@
   private final String mUfsRoot = mMasterConf.get(Constants.UNDERFS_ADDRESS);
   private final UnderFileSystem mUfs = UnderFileSystem.get(mUfsRoot, mMasterConf);
 
-  private AlluxioMasterRestServiceHandler() {} // prevent instantiation
```

Comment:
here is my thought, let me know if they make sense to you. This is not a util class (which is supposed to expose static function), it has member variables. So by default it should have public constructor --- otherwise external classes cannot create instances of this class. In other words, adding the private constructor is a bug. I feel it is wired to comment on why we need a public constructor here. what do you think?